src/UI/Component/Layout/Page/Factory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace ILIAS\UI\Component\Layout\Page;
+/**
+ * This is the factory for page-layout components
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The Notification Center concentrates the visualization of _any_
+     *     event-notification into one expandable glyph.
+     *     This unification removes the multitude of dedicated notification-icons
+     *     in favor of visual cleanliness as well as providing a designated
+     *     location for further extensions.
```

Comment:
I once heard the nice definition of "The notification center" offers possible "reactions" to notifications of the system for the user (e.g. new Mail) while the Cockpit offers action (e.g. open a course). Can we add that here somehow?